libvast/src/system/node.cpp

```diff
@@ -27,192 +29,207 @@
 #include "vast/concept/printable/stream.hpp"
 #include "vast/concept/printable/to_string.hpp"
 #include "vast/concept/printable/vast/json.hpp"
+#include "vast/detail/assert.hpp"
 #include "vast/json.hpp"
 #include "vast/logger.hpp"
 
 #include "vast/system/accountant.hpp"
 #include "vast/system/consensus.hpp"
-#include "vast/system/node.hpp"
 #include "vast/system/spawn.hpp"
 
-using namespace std::string_literals;
+using std::string;
+
 using namespace caf;
 
-namespace vast {
-namespace system {
+namespace vast::system {
 
 namespace {
 
-using node_ptr = stateful_actor<node_state>*;
+// Local commands need access to the node actor.
+thread_local node_actor* this_node;
 
-void stop(node_ptr self) {
-  self->send_exit(self, exit_reason::user_shutdown);
+// Convenience function for wrapping an error into a CAF message.
+auto err(ec code, string msg) {
```

Comment:
I don't find `err` to be a readable function name. How about calling this `fail`? Also short and much more readable.

Reply:
`make_error_msg ` works for me, `fail` was just an example for something more readable.

Reply:
Cool, I already renamed it `make_error_msg`. 👍